examples/passing/PartialTCO.purs

```diff
@@ -0,0 +1,13 @@
+module Main where
+
+import Prelude
+import Control.Monad.Eff.Console (log)
```

Comment:
Should this be `Effect.Console`? How is this test passing? 😆

Reply:
This test is in the wrong directory. It should be in `tests/purs/passing`. I think this is because the tests were moved sometime after the point you made this PR.

Reply:
Yikes! The perils of rebasing :)

I've pushed a fix, thanks.